lib/Parse/DNS/Zone.pm

```diff
@@ -408,6 +413,42 @@ sub _load_zonefile {
 	return do { local $/; <$zonefh> }; # slurp
 }
 
+sub _parse_time_unit {
+	my $string = shift;
+
+	my $time_units_regex = qr/^
+			(?: (?=\d+(?: w|d|h|m|s))
+			(\d+w)? (\d+d)? (\d+h)? (\d+m)? (\d+s)? ) | (\d+)
+		$/xi;
+	my @parts = ($string =~ $time_units_regex);
+
+	my %multi_for_unit = (
+		s => 1,
+		m => 60,
+		h => 3600,
+		d => 86400,
+		w => 604800,
+	);
+
+	my $seconds = 0;
+
+	for my $part (@parts) {
+		next
+			unless $part;
+		my ($value, $unit) = $part =~ /^(\d+)(\D+)?$/;
+		$unit = 's'
+			unless $unit;
+		# from Bind reference manual:
+		# Both TTL-style and ISO 8601 duration formats are case-insensitive.
```

Comment:
I kind of think the comment is superfluous, especially in combination with tests covering this. At least remove mentions of ISO 8601, as we don't support it. (Fixing that would fix the line length as well. :).)